src/themes/money/theme.json

```diff
@@ -2760,6 +2760,54 @@
           "border": "0.08em solid #DFDFE1 !important"
         }
       }
+    },
+    "navigation-dropdown": {
+      "button": {
+        "boxShadow": "#DFDFE1 0px 0px 0px 1px",
+        "borderRadius": "4px"
+      },
+      "button:focus, button:focus-visible": {
+        "borderColor": "plum"
+      },
+      "title": {
+        "variant": "compounds.billboard.primaryContent.p",
+        "color": "grey-0",
+        "mb": "sm"
+      },
+      "ul": {
+        "width": "100%",
+        "maxWidth": "400px",
+        "maxHeight": "272px",
+        "overflow": "scroll",
+        "py": "sm",
+        "px": 0,
+        "backgroundColor": "#ffffff",
+        "minWidth": "160px",
+        "boxShadow": "0px 8px 16px 0px rgba(0,0,0,0.2)",
+        "mt": 0
+      },
+      "ul:hover": {
+        "cursor": "pointer"
+      },
+      "ul > li": {
```

Comment:
Why are we doing this `"ul > li"` instead of
```json
"ul": {
  "&:hover": {
    ...
  }
  "li": {
    ...
  }
}
```

Reply:
Nice suggestion. Updated the styling.